docs/magento2/search-engine.mdx

```diff
@@ -360,6 +361,45 @@ query Search {
 If you are using the default theme or the Chocolatine theme, the search bar
 should now be visible.
 
+## Attraqt recommendations
+
+> _Since version `2.19`_
+
+Front-Commerce provides a loaders to retrieve recommendations from
+[Attraqt's Widgets](https://attraqt.gitbook.io/developer-documentation/xo-recommendations/readme-first).
+
+### Front-Commerce configuration
+
+On Front-Commerce side, you need to enable the Attraqt recommendation module by
+making the following changes in your .front-commerce.js file:
+
+```js title=".front-commerce.js"
+module.exports = {
+  // highlight-next-line
+  modules: ["./node_modules/front-commerce/modules/recommendations-attraqt"],
+  serverModules: [
+    { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
```

Comment:
Does this actually exist? otherwise I will not add it here as to avoid confusion.

Reply:
I don't think it exists actually, I took the same base as the snippets above in this file. I'll fix it !

_Note: It seems the "FrontCommerceCore" module has been already used a good number of times throughout the doc already 😕_

Reply:
I think it should be removed or fixed (in a seperate mr)